site/_posts/2017-02-20-februarys-final-weeks-work.md

```diff
@@ -0,0 +1,56 @@
+* ManageIQ Content saw [9 PRs][manageiq-content PRs merged] merged (awarded most generic! no really, checkout those prs!) :3rd_place_medal: 
+* ManageIQ Service UI saw [9 PRs][manageiq-ui-service PRs merged] merged (awarded most housekeeping!) :4th_place_medal:
+* Amazon provider had [4 PRs][manageiq-providers-amazon PRs merged] merged (awarded most resilient!) :5th_place_medal:
+* Azure provider had [3 PRs][manageiq-providers-azure PRs merged] merged (awarded most resilient!) :6 th_place_medal:
+
+But this :video_game: we play is not about the :trophy:'s, :taco:'s or :+1: its about the change, once such type of change being the Improvements!
+## Improved
+Every line of code contributed across the ManageIQ repo collection is an improvement, but since neither of us have the attention to look at each of the 169 improvements from last week, here are a few highlights:
+* [Marcel](https://github.com/durandom) refreshed the workings of MIQ core credentials in support of Ansible Tower here [Ansible refresh inventory credentials](https://github.com/ManageIQ/manageiq/pull/13834)
+* [Milan](https://github.com/mzazrivec) introduced into MIQ Core new menu sections to display Ansible Playbooks [UI for Ansible Tower Playbooks & Repositories](https://github.com/ManageIQ/manageiq-ui-classic/pull/283)
+
+MIQ Core experienced a pair of key performance improvements:
+* [Šimon](https://github.com/isimluk) introduced changes to speed up reports, [Drop the precision calculation](https://github.com/ManageIQ/manageiq/pull/13899) 
+* [Libor](https://github.com/lpichler), for the second week running, has contributed to improved performance with [Use eager_load for extra_resources](https://github.com/ManageIQ/manageiq/pull/13904)
+
+## Fixed
+Sometimes an improvement is of the type two steps back.  In this case, a followup improvement of the type two steps forward is required, :bug: fixes! 
+The MIQ Service UI repo was brought a fix of this nature by [Chris](https://github.com/chalettu), [Fix for refresh of SUI $state.FeatureNav issue](https://github.com/ManageIQ/manageiq-ui-service/pull/496/files) which eliminated role based errors experienced when reloading the UI.
+In MIQ Classic repo broken automate icons [Fix broken Automate icons](https://github.com/ManageIQ/manageiq-ui-classic/pull/351) were fixed by [Eric](https://github.com/epwinchell).
+MIQ Core experienced the most hotly discussed bug fix (42 comments :eyes::feelsgood:) [Create a hawkular client for partial endpoints](https://github.com/ManageIQ/manageiq/pull/13814) authored by [Mooli](https://github.com/moolitayer)
+
+## New
+Hold the :phone:'s **newly introduced** REST API endpoints for service templates! This deeply pondered pr (Nov 2016  :scream_cat: ) [create service template REST api](https://github.com/ManageIQ/manageiq/pull/125940) was slain to completion by [Jillian](https://github.com/ManageIQ/manageiq/pull/12594).  
```

Comment:
@AllenBW - PR #125940 resulted in a `404`, also not sure if you meant to go directly to Jillian's github page or go to PR #12594 in the last url labeled `Jillian`